Sources/Perception/PerceptionRegistrar.swift

```diff
@@ -207,24 +206,13 @@ extension PerceptionRegistrar: Hashable {
 
   var isInSwiftUIBody: Bool {
     for callStackSymbol in Thread.callStackSymbols {
+      let mangledSymbol = callStackSymbol.utf8
+        .drop(while: { $0 != .init(ascii: "$") })
+        .prefix(while: { $0 != .init(ascii: " ") })
       guard
-        let symbol = callStackSymbol.split(separator: " ").dropFirst(3).first
-      else {
-        continue
-      }
-      guard
-        symbol.utf8.first == .init(ascii: "$")
-      else {
-        continue
-      }
-      guard
-        let demangled = String(symbol).demangled
-      else {
-        continue
-      }
-      guard
-        demangled.hasPrefix("protocol witness for SwiftUI.View.body.getter : ")
-          || demangled.contains(" SwiftUI.") && demangled.contains("body.getter : some")
+        let demangled = String(Substring(mangledSymbol)).demangled,
+        demangled.contains("body.getter : "),
```

Comment:
I had to pretty significantly weaken this condition in order for this simple test to emit a warning:

```swift
struct FeatureView: View {
  let model = Model()
  var body: some View {
    Text(self.model.count.description)
  }
}
```

It's probably not very common, but it seems that if there's just a single, simple view at the root of the `body` then the call stack is simplified a bit and there's no mention of `SwiftUI.` or even `some`. The generics are made concrete in the call stack.